modules/search/activesearch.py

```diff
@@ -0,0 +1,50 @@
+"""
+OWASP Maryam!
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+any later version.
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""
+
+
+meta = {
+	'name': 'Active Search Results',
+	'author': 'Kaushik',
+	'version': '0.1',
+	'description': 'Active Search Results (ASR) is an independent Internet Search \
+		Engine using a proprietary page ranking technology with Millions of \
+		popular Web sites indexed.',
+	'sources': ('activesearch'),
+	'options': (
+		('query', None, True, 'Query string', '-q', 'store', str),
+		('limit', 15, False, 'Max result count (default=15)', '-l', 'store', int),
+	),
+	'examples': ('activesearch -q <QUERY> -l 15 --output',)
+}
+
+def module_api(self):
+	query = self.options['query']
+	limit = self.options['limit']
+	run = self.activesearch(query, limit)
+	run.run_crawl()
+	links = run.links
+	output = {'links': links}
+
+	# To get titles along with links
```

Comment:
remove these.
Use the links_with_data method.

Reply:
thats what I initially did but google only outputted links so I used this

Reply:
Did you use google?

Reply:
i only looked at it

Reply:
Google doesn't show because it doesn't scrape the title and description. so use the links_with_data

Reply:
Yep done.